config/config.go

```diff
@@ -25,9 +25,11 @@ type Config struct {
 	IdentityPublicKey string `yaml:"identity-public-key"`
 	IdentityLocation  string `yaml:"identity-location"`
 	// The identity API is optional
-	IdentityAPIURL      string `yaml:"identity-api-url"`
-	IdentityAPIUsername string `yaml:"identity-api-username"`
-	IdentityAPIPassword string `yaml:"identity-api-password"`
+	IdentityAPIURL        string `yaml:"identity-api-url"`
+	IdentityAPIUsername   string `yaml:"identity-api-username"`
+	IdentityAPIPassword   string `yaml:"identity-api-password"`
+	IdentityAPIPrivateKey string `yaml:"identity-api-private-key"`
```

Comment:
I agree that would be better, but it didn't really fit in wiaht that was already here, so I assumed it couldn't be a *bakery.Key for some reason.